VideoPlayer/Android/testplayerplugin-android/src/main/java/com/applicaster/player/plugins/testplayerplugin_android/TestPlayerAdapter.java

```diff
@@ -0,0 +1,186 @@
+        videoView = new VideoView(context);
+    }
+
+    /**
+     * initialization of the player plugin configuration with a Map params.
+     * the params taken from res/raw/plugin_configurations.json
+     *
+     * @param params
+     */
+    @Override
+    public void setPluginConfigurationParams(Map params) {
+        super.setPluginConfigurationParams(params);
+    }
+
+    /**
+     * get the first playable attached to the player
+     *
+     * @return playable
+     */
+    @Override
+    public Playable getFirstPlayable() {
+        return super.getFirstPlayable();
+    }
+
+    /**
+     * return the player plugin configuration
+     *
+     * @return Player configuration.
+     */
+    @Override
+    public Map getPluginConfigurationParams() {
+        return super.getPluginConfigurationParams();
+    }
+
+    @Override
+    public List<Playable> getPlayableList() {
+        return super.getPlayableList();
+    }
+
+    /**
+     * start the player in fullscreen with configuration.
+     *
+     * @param playableConfiguration player configuration.
+     * @param requestCode           request code if needed - if not send NO_REQUEST_CODE instead.
+     * @param context
+     */
+    @Override
+    public void playInFullscreen(PlayableConfiguration playableConfiguration, int requestCode, Context context) {
+        super.playInFullscreen(playableConfiguration, requestCode, context);
+        Intent intent = new Intent(context, TestPlayerActivity.class);
+        if (getFirstPlayable() != null) {
+            intent.putExtra(KEY_PLAYABLE, getFirstPlayable());
+            intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP);
+            context.startActivity(intent);
+        }
+    }
+
+    /**
+     * Add the player into the given container.
+     *
+     * @param viewGroup The container the player should be added to
+     */
+    @Override
+    public void attachInline(ViewGroup viewGroup) {
+        super.attachInline(viewGroup);
+        if (videoView != null) {
+            viewGroup.addView(videoView);
+        }
+    }
+
+    /**
+     * Remove the player from it's container.
+     *
+     * @param viewGroup The container that the player is attached to
+     */
+    @Override
+    public void removeInline(ViewGroup viewGroup) {
+        super.removeInline(viewGroup);
+        if (videoView != null) {
+            viewGroup.removeView(videoView);
+        }
+    }
+
+
+    /**
+     * Starts playing the inline player.
+     */
+    @Override
+    public void playInline() {
+        super.playInline();
+        if (getFirstPlayable() != null) {
+            videoView.setVideoURI(Uri.parse(getFirstPlayable().getContentVideoURL()));
+            videoView.start();
+            videoView.setOnPreparedListener(new MediaPlayer.OnPreparedListener() {
+                @Override
+                public void onPrepared(MediaPlayer mediaPlayer) {
+                    mediaPlayer.start();
```

Comment:
Same as before. Why do we need it in 2 different places?

Reply:
One is for the inline, one is for the fullscreen